src/components/plots/barPlot.ts

```diff
@@ -142,6 +159,31 @@ export module Plots {
       }
     }
 
+    public addDataset(dataset: Dataset) {
+      if (this._isSetup) {
+        var labelArea = this._renderArea.append("g").classed(Bar._LABEL_AREA_CLASS, true);
```

Comment:
Maybe we should have some sort of `_setUpDrawer()` call that gets invoked in `Plot`'s `addDataset()`?